.github/workflows/flutter.yml

```diff
@@ -59,13 +64,19 @@ jobs:
           env:
             SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
           run: |
+            flutter config --enable-windows-desktop
+            flutter config --enable-macos-desktop
+            flutter config --enable-linux-desktop
             cd flutter/example
             TARGET=${{ matrix.target }}
             flutter pub get
             case $TARGET in
             ios)
             flutter build ios --release --no-codesign
             ;;
+            macos)
+            flutter build macos --release
```

Comment:
build web is missing `--release`
wondering if `--release` is the default value and it'd be easier to just removing from all of them